benchexec/tools/ultimate.py

```diff
@@ -204,113 +188,136 @@ def _requires_ultimate_data(self, executable):
         # all versions before 0.1.24 do not require ultimatedata
         return not (int(major) == 0 and int(minor) < 2 and int(patch) < 24)
 
-    def cmdline(self, executable, options, tasks, propertyfile=None, rlimits=None):
-        if rlimits is None:
-            rlimits = {}
-
-        self._uses_propertyfile = propertyfile is not None
-        if _OPTION_NO_WRAPPER in options:
-            # do not use old wrapper script even if property file is given
-            self._uses_propertyfile = False
-            propertyfile = None
-            options.remove(_OPTION_NO_WRAPPER)
+    def cmdline(self, executable, options, task, resource_limits):
+        data_model_param = get_data_model_from_task(
+            task, {ILP32: "32bit", LP64: "64bit"},
```

Comment:
```suggestion
            task, {ILP32: "32bit", LP64: "64bit"}
```

The version of black used by us would otherwise reformat this. I would actually have committed this myself but could not push to the PR branch, so please apply this suggestion.